gha/py-dependencies/action.yaml

```
@@ -55,27 +59,31 @@ runs:
         ./scripts/github/install-virtualenv.sh
         PIP_VERSION=$(grep '^PIP_VERSION' Makefile | awk '{print $3}')
         echo "::set-output name=pip-version::${PIP_VERSION}"
-        ${VIRTUALENV_DIR}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
+        ${{env.VIRTUALENV_DIR}}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
 
     - name: Install StackStorm requirements
       # this should run in the st2 checkout
+      shell: bash
       working-directory: st2
       env:
         VIRTUALENV_DIR: "~/virtualenv"
```

OK. Since tilde expansion isn't happening like I thought, then maybe we can do this instead:
```suggestion
        VIRTUALENV_DIR: ${{ env.HOME }}/virtualenv
```

To do that, we need to add a step at the beginning of the job like this:
```yaml
- name: add HOME to env context
  shell: bash
  run: echo ::set-env name=HOME::$HOME
```

---

I just pushed this into change to the upstream gha branch.